src/DynamoApplications/StartupUtils.cs

```diff
@@ -43,7 +43,7 @@ internal class CMDLineOptions
         [Option('i', "Import", Required = false, HelpText = "Instruct Dynamo to import an assembly as a node library.This argument should be a filepath to a single.dll" +
             " - if you wish to import multiple dlls - list the dlls separated by a space: -i 'assembly1.dll' 'assembly2.dll'")]
         public IEnumerable<String> ImportedPaths { get; set; } = new List<string>();
-        [Option('g', "GeometryPath", Required = false, HelpText = "relative or absolute path to a directory containing ASM. When supplied, instead of searching the hard disk for ASM, it will be loaded directly from this path.")]
+        [Option('gp', "GeometryPath", Required = false, HelpText = "relative or absolute path to a directory containing ASM. When supplied, instead of searching the hard disk for ASM, it will be loaded directly from this path.")]
```

Comment:
does this align with the net 48 options? I think it uses a different parsing library? FYI @pinzart90 will know.

Reply:
I kind of remember this library only supports single letter flags.

Reply:
`D:\a\Dynamo\Dynamo\Dynamo\src\DynamoApplications\StartupUtils.cs(46,17): error CS1012: Too many characters in character literal`

Reply:
In that case we need new character to avoid duplication with https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoApplications/StartupUtils.cs#L41

Reply:
@avidit is there an overload for option where you can supply only the full `verb` and not have an associated character?

Reply:
Not sure